x/ibc/09-localhost/client_state.go

```diff
@@ -0,0 +1,270 @@
+package localhost
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/types"
```

Comment:
this is the standard way we use to import types
```suggestion
	sdk "github.com/cosmos/cosmos-sdk/types"
```